src/components/resource/edit-managed-title/resource-edit-managed-title.js

```diff
@@ -246,25 +260,25 @@ export default class ResourceEditManagedTitle extends Component {
   }
 
   render() {
-    let {
+    const {
       model,
       proxyTypes,
-      initialValues,
     } = this.props;
 
-    let {
+    const {
       showSelectionModal,
       managedResourceSelected,
       sections,
+      initialValues,
     } = this.state;
 
-    let isSelectInFlight = model.update.isPending && 'isSelected' in model.update.changedAttributes;
+    const isSelectInFlight = model.update.isPending && 'isSelected' in model.update.changedAttributes;
```

Comment:
this statement is a bit overcomplicated 
At least add brackets in  model.update.isPending && ( 'isSelected' in model.update.changedAttributes') ;